### PR TITLE
ensure package.json exports proper file + add a test case to check it.

### PR DESCRIPTION
**Pull request by knksmith57**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+describe('$', function () {
+  before(function () {
+  });
+
+  after(function () {
+  });
+
+  it('should export properly', function () {
+    should.exist(require('..'));
```
Comment (knksmith57) [inline test/index.js]: I'm not sure if using something besides `.exist()` would be better here. Seems like this is a good enough check to me though.

Comment (dthree) [inline test/index.js]: LGTM

**Comment by dthree** [comment]
>  Coverage decreased (-0.0%) to 97.733% 

? Haha

---

Thanks a million. :+1:

**Comment by knksmith57** [comment]
> Coverage decreased (-0.0%) to 97.733%

Lol, coveralls is weird sometimes.

No problem and thanks for the quick turn around! I'm excited to play with the package.

**Comment by dthree** [comment]
Okay great! Hope you find it helpful!

**Pull request merged**